Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/english/TimexConvertEnglish.java

```diff
@@ -55,28 +55,28 @@ public static String convertTimexToString(TimexProperty timex) {
     public static String convertTimexSetToString(TimexSet timexSet) {
         TimexProperty timex = timexSet.getTimex();
         if (timex.getTypes().contains(Constants.TimexTypes.DURATION)) {
-            return String.format("every %s", TimexConvertEnglish.convertTimexDurationToString(timex, false));
+            return TimexConstantsEnglish.Every.concat(TimexConvertEnglish.convertTimexDurationToString(timex, false));
```

Comment:
This won't work, as there is no whitespace between `EVERY` and the result of `convertTImexDurationToString`.

Change it to `String.format`